src/test/ui/coherence/const-generics-orphan-check-ok.rs

```diff
@@ -0,0 +1,28 @@
+// check-pass
+// aux-build:trait-with-const-param.rs
+extern crate trait_with_const_param;
+use trait_with_const_param::*;
+
+// Trivial case, const param after local type.
```

Comment:
Wrong description? "Impl for local type"?

Reply:
in the substs of that impl trait ref we have `[Local1, N, T]`, so const param after local type holds :grin: 

want me to change that comment, don't think that "impl for local type" is the right descr because the same also holds for `impl<const N; usize, T> OtherTrait<Local1, N, T> for ForeignTy`